Python/Product/PythonTools/visualstudio_py_repl.py

```diff
@@ -14,6 +14,11 @@
 
 from __future__ import with_statement
 
+# This module MUST NOT import threading in global scope. This is because in a direct (non-ptvsd)
+# attach scenario, it is loaded on the injected debugger attach thread, and if threading module
+# hasn't been loaded already, it will assume that the thread on which it is being loaded is the
+# main thread. This will cause issues when the thread goes away after attach completes.
+

```

Comment:
To implement the "connect REPL" command for the debug REPL. Unfortunately, that cannot just import the module locally when and where it needs it, because in direct attach it's not on the import path - so PyDebugAttach has to manually load and inject it (and util).